### PR TITLE
[Example] Attach training functionality @open sesame 08/04 20:43

### PR DESCRIPTION
**Pull request by zhoonit**
**Changes proposed in this PR:**
- Attach nntrainer api for training

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

### CODE REVIEW

File: Applications/Tizen_native/CustomShortcut/src/data.c [hunk 1]
```diff
@@ -237,9 +246,59 @@ int data_extract_feature(appdata_s *ad, const char *dst, bool append) {
...
+    _E("Error closing file");
+    return;
+  }
+
+  _D("model conf path: %s", model_conf_path);
+
+  status = ml_train_model_construct_with_conf(model_conf_path, &model);
+  if (status != ML_ERROR_NONE) {
+    _E("constructing trainer model failed %d", status);
+    goto CLEAN_UP;
```
Comment (kparichay) [inline Applications/Tizen_native/CustomShortcut/src/data.c:281]: simply return?

File: Applications/Tizen_native/CustomShortcut/src/data.c [hunk 2]
```diff
@@ -237,9 +246,59 @@ int data_extract_feature(appdata_s *ad, const char *dst, bool append) {
...
+  }
+
+  status = ml_train_model_destroy(model);
+  if (status != ML_ERROR_NONE) {
+    _E("Destryoing model failed %d", status);
+  }
 
-  ml_train_model_construct(model);
+CLEAN_UP:
   ml_train_model_destroy(model);
```
Comment (kparichay) [inline Applications/Tizen_native/CustomShortcut/src/data.c:302]: log return valus of `ml_train_model_destroy` if this also fails?

File: Applications/Tizen_native/CustomShortcut/src/data.c (hunk 1 above)
Comment (zhoonit) [reply Applications/Tizen_native/CustomShortcut/src/data.c]: Fixed

File: Applications/Tizen_native/CustomShortcut/src/data.c (hunk 2 above)
Comment (zhoonit) [reply Applications/Tizen_native/CustomShortcut/src/data.c]: It is done  Thanks

**Comment by jijoongmoon** [comment]
rebase plz

**Comment by zhoonit** [comment]
> rebase plz

rebased

**Pull request merged**